tests/test_interface.py

```diff
@@ -16,6 +30,16 @@ def render_board(self, board_state):
             return out
 
 
+@pytest.fixture
+def valid_interface():
```

Comment:
Perhaps the fixture name ought to reflect the changes to what is being returned?